android/assets/jsons/Civ V - Gods & Kings/Nations.json

```diff
@@ -595,8 +595,8 @@
 		"startBias": ["Avoid [Tundra]"],
 		"preferredVictoryType": "Domination",
 
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "Askia, leader of the Songhai people and greatest of all, may God watch your path towards glory! Songhai was under the rule of the mighty West African state of Mali until the middle of the 14th century. Great King Sunni Ali Ber fought for independance, and showed to all Songhai's power, by conquering territories and repelling many foes who sought to destroy the rising kingdom. Ultimately, conquest of Timbuktu and Jenne wealthy cities made from Songhai an empire with enough economic power to survive for a century, until the empire bow down before foes with advanced technology - muskets against spearmen.",
```

Comment:
Ultimately, conquest of the wealthy cities of Timbuktu and Jeanne made Songhai
until the empire bowed down